aodn_cloud_optimised/config/dataset/mooring_timeseries_realtime_qc.json

```diff
@@ -0,0 +1,327 @@
+{
+  "dataset_name": "mooring_timeseries_realtime_qc",
+  "logger_name": "mooring_timeseries_realtime_qc",
+  "cloud_optimised_format": "parquet",
+  "run_settings": {
+    "paths": [
+      {
+        "s3_uri": "s3://imos-data/IMOS/ANMN/NRS/NRSMAI/real-time",
+        "filter": [
+          ".*\\.nc"
+        ],
+        "year_range": []
+      }
+    ],
+    "cluster": {
+      "mode": "local",
+      "restart_every_path": false
+    },
+    "clear_existing_data": true,
+    "raise_error": false,
+    "coiled_cluster_options": {
+      "n_workers": [
+        1,
+        20
+      ],
+      "scheduler_vm_types": "m7i-flex.large",
+      "worker_vm_types": "m7i-flex.large",
+      "allow_ingress_from": "me",
+      "compute_purchase_option": "spot_with_fallback",
+      "worker_options": {
+        "nthreads": 4,
+        "memory_limit": "8GB"
+      }
+    },
+    "batch_size": 5,
+    "force_previous_parquet_deletion": false
```

Comment:
@mhidas this is the bit which sets to overwritte data. has to be set to true in your config